runelite-client/src/main/java/net/runelite/client/util/OSType.java

```diff
@@ -36,13 +36,7 @@
 
 	private static OSType cached;
 
-	/**
-	 * Uses the os.name property to detect the os type.
-	 *
-	 * @return An {@link OSType} for the machine's OS.
-	 */
-	public static OSType getOSType()
-	{
+	static {
 		if (OSType.cached == null)
```

Comment:
No need to add null check here, the static ctor will be called only once, so the if is always true.